subworkflows/local/align_star/main.nf

```diff
@@ -1,21 +1,22 @@
 //
 // Alignment with STAR
 //
-
-include { STAR_ALIGN              } from '../../../modules/nf-core/star/align'
-include { STAR_ALIGN_IGENOMES     } from '../../../modules/local/star_align_igenomes'
+include { SENTIEON_STARALIGN as SENTIEON_STAR_ALIGN} from '../../../modules/nf-core/sentieon/staralign/main'
+include { STAR_ALIGN } from '../../../modules/nf-core/star/align'
+include { STAR_ALIGN_IGENOMES } from '../../../modules/local/star_align_igenomes'
 include { BAM_SORT_STATS_SAMTOOLS } from '../../nf-core/bam_sort_stats_samtools'
 
 workflow ALIGN_STAR {
     take:
-    reads               // channel: [ val(meta), [ reads ] ]
-    index               // channel: [ val(meta), [ index ] ]
-    gtf                 // channel: [ val(meta), [ gtf ] ]
+    reads // channel: [ val(meta), [ reads ] ]
+    index // channel: [ val(meta), [ index ] ]
+    gtf // channel: [ val(meta), [ gtf ] ]
     star_ignore_sjdbgtf // boolean: when using pre-built STAR indices do not re-extract and use splice junctions from the GTF file
-    seq_platform        // string : sequencing platform
-    seq_center          // string : sequencing center
-    is_aws_igenome      // boolean: whether the genome files are from AWS iGenomes
-    fasta               // channel: /path/to/fasta
+    seq_platform // string : sequencing platform
```

Comment:
Can we have the comment alignments back? :-P

Reply:
we can, but blame the auto-formatting :shake_fist:

Reply:
43abdc72